src/tools/miri/src/shims/windows/foreign_items.rs

```diff
@@ -804,8 +811,9 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
                     );
                 }
                 // This function looks and behaves excatly like miri_start_unwind.
+                // TODO: this is an unwind: true, would it still be correct after it is changed to Conv::X86Stdcall?
```

Comment:
`Conv` doesn't contain the unwind information -- it doesn't have to.

However, since this was `ExternAbi::C` before, I think it should be `Conv::C` now.